services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/DelegatingDeviceManagementHttpEndpoint.java

```diff
@@ -135,6 +143,39 @@ private void doGetDevice(final RoutingContext ctx) {
             .onComplete(s -> span.finish());
     }
 
+    private void doSearchDevices(final RoutingContext ctx) {
+        final Span span = TracingHelper.buildServerChildSpan(
+                tracer,
+                TracingHandler.serverSpanContext(ctx),
+                SPAN_NAME_SEARCH_DEVICES,
+                getClass().getSimpleName()).start();
+
+        final Future<String> tenantId = getRequestParameter(ctx, PARAM_TENANT_ID,
+                getPredicate(config.getTenantIdPattern(), false));
```

Comment:
I do not think that we need to check the tenant ID here as we are merely using it as a query parameter, right? We do not need to prevent creation of a tenant with an illegal ID here,

Reply:
Ok. Since tenant ID here is a _path_ parameter, I will get it directly using `ctx.request().getParam(PARAM_TENANT_ID)`